core/src/main/java/org/mskcc/oncotree/model/TumorType.java

```diff
@@ -262,8 +269,8 @@ public String toString() {
         sb.append("  color: ").append(color).append("\n");
         sb.append("  name: ").append(name).append("\n");
         sb.append("  mainType: ").append(mainType).append("\n");
-        sb.append("  NCI: ").append(NCI).append("\n");
-        sb.append("  UMLS: ").append(UMLS).append("\n");
+        sb.append("  NCI: ").append(StringUtils.join(NCI, ",")).append("\n");
```

Comment:
Nope you were not wrong.  The API outputs this now: 
"NCI": [
                "C3246",
                "C7052"
              ],
              "UMLS": [
                "C0026948",
                "C1266121"
              ]